package.json

```diff
@@ -46,5 +46,6 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^4.3.1",
     "mocha": "^8.4.0"
-  }
+  },
+  "eslintIgnore": ["sign.js", "main.js"]
```

Comment:
are these the JS files you are ignoring?  I think you can just ignore a folder instead of just individual files to make it easier